user_guide/intro_users_and_benefits.rst

```diff
@@ -0,0 +1,163 @@
+.. _oro-benefits:
+
+OroCRM Users and Their Benefits
+===============================
+
+This chapter of the guide explains the most common OroCRM user roles in order to help you understand which sections of 
+this user guide best match your work duties.
+
+Something for Everyone
+----------------------
+
+Collect and Process information easily
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+OroCRM handles critical CRM tasks such as:
+
+- Collect and aggregate information related to customers, sales, marketing campaigns, sales pipeline, and high level 
+  business performance. 
+
+- Organize the information in a way that is both meaningful and efficient
+
+- Ensure easy, convenient, and safe access to information for all the stakeholders with customizable access and 
+  permission settings. 
+
+- Create workflows that mirror or improve upon existing business processes as well as provide an overhead view of that 
+  pipeline  
+
+
+Please refer to the :ref:`Data Management <user-guide-data-management-basics>` section to understand how information is 
+represented and can be processed within OroCRM.
+
+
+
+Get a 360-Degree view of your customers activity
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Centralizing information about your customers and their activities is at the heart of OroCRM. In order to learn how 
+customer accounts and contacts are managed in OroCRM, please refer to the 
+:ref:`Common Features <user-guide-common-features>` section.
+
+
+Organize information and make informed decisions
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+OroCRM’s business intelligence tools will help to organize data and improve the decision making process:
+
+- Use predefined widgets to create informative dashboards to display metrics, pipelines, and gain quick access to 
+  regularly used activities
+
+- Analyze and visualize historical data and trends with reports
+
+- Narrow down the exact segment of data you’re looking for with segmentation 
+
+Please refer to the :ref:`Business Intelligence Tools section <business-intelligence>` of the User Guide of this 
+guide to learn more about how OroCRM can be used to analyze information.
+
+
+Communicate and Delegate
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+Any CRM user can benefit from OroCRM’s productivity tools which allow any authorized user to:
+
+- Create calendar events for yourself or other users
+
+- Create templates with merge fields or write custom emails in the fly
+
+- Log calls
+
+- Assign tasks to yourself or to others 
+
+- Make notes and comments on different entities within OroCRM
+
+See the :ref:`Productivity Tools section <user-guide-productivity-tools>` of the User Guide to learn more about how OroCRM can help 
+to improve communication and task management within the company.
```

Comment:
Edit: See the :ref:`Productivity Tools section <user-guide-productivity-tools>` of the User Guide to learn more about how OroCRM can help to improve communication and task management within your company.